remote-dependency-templates/README.md

```diff
@@ -1,11 +1,87 @@
-## Usage
+# Using dependency templates
 - List dependency `<id>` as a remote dependency in `application.yml`, with one of the supported types
   - Populate required `template-config` fields (defined in the README for each type)
 - List any dependency-specific service data under `dependency-data.<id>` in a service's `service.yml`
 
-## Terraform variables //TODO
+# Building dependency templates
+- Create a directory with a unique name. This will be the dependency's `type`.
+- The directory should contain the following three files: requirements.yml, dependency.tf and README.md. Each is explained in detail below. (see existing templates for example)
```

Comment:
This should be updated to mention the modules folder. Also please put the filenames in backticks.